src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using Teams.Data;
+using Teams.Models;
+using Teams.Security;
+
+namespace Teams.Services
+{
+    public class ManageTeamsMembersService : IManageTeamsMembersService
+    {
+        private readonly ICurrentUser _currentUser;
+        private readonly IRepository<Team, int> _teamRepository;
+        private readonly IRepository<TeamMember, int> _memberRepository;
+
+        public ManageTeamsMembersService(IRepository<Team, int> teamRepository, IRepository<TeamMember, int> memberRepository, ICurrentUser currentUser)
+        {
+            _currentUser = currentUser;
+            _teamRepository = teamRepository;
+            _memberRepository = memberRepository;
+        }
+        public async Task<bool> RemoveAsync(int team_id, string member_id)
+        {
+            Team team = _teamRepository.GetByIdAsync(team_id).Result;
```

Comment:
var team
why Result?
It is not first time, please read about Tasks and async/await operations.
If you can not get something, please write about this

Reply:
Also read about *Include* operation, you can get team and check exists this member inside member list in team or not